src/custom/utils/operator.ts

```diff
@@ -8,7 +8,7 @@ import { registerOnWindow } from './misc'
  *    https://protocol-rinkeby.dev.gnosisdev.com/api/
  */
 const API_BASE_URL: Partial<Record<ChainId, string>> = {
-  [ChainId.MAINNET]: 'https://protocol.gnosis.io/api/v1',
+  [ChainId.MAINNET]: 'https://protocol-mainnet.dev.gnosisdev.com/api/v1',
```

Comment:
@W3stside why this change? if this is temporal. Please make sure there's a TODO and falls under your radar

Maybe this URLs should be envs? so we can change it in travis. We leave the mainnet by default, but we make travis use dev mainnet for DEV and PR branches?

Reply:
@anxolin this thread is relevant: https://gnosisinc.slack.com/archives/C6Z2XNL5Q/p1608205597200800

Reply:
but yes I'll make a note of it